samples/openthread/ncp/prj.conf

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+# ===============================
+# KERNEL OPTIONS
```

Comment:
Let's keep the comment style in `prj.conf` uniform with other samples in NCS - I don't think such bulky headers are needed. If you need a separator, a single line comment is enough.

Reply:
Done